op-node/rollup/derive/calldata_source.go

```diff
@@ -111,12 +111,12 @@ func DataFromEVMTransactions(config *rollup.Config, batcherAddr common.Address,
 		if to := tx.To(); to != nil && *to == config.BatchInboxAddress {
 			seqDataSubmitter, err := l1Signer.Sender(tx) // optimization: only derive sender if To is correct
 			if err != nil {
-				log.Warn("tx in inbox with invalid signature", "index", j, "err", err)
+				log.Warn("tx in inbox with invalid signature", "index", j, "txHash", tx.Hash().Hex(), "err", err)
```

Comment:
```suggestion
				log.Warn("tx in inbox with invalid signature", "index", j, "txHash", tx.Hash(), "err", err)
```

`common.Hash` has a nice `TerminalString` implementation which makes it automatically log nicely.